### PR TITLE
Add insertion script

### PR DESCRIPTION
**Pull request by togimoto**
The data from json has been inserted into the database. Please see the Neon database.

### CODE REVIEW

File: db-docs/dbdiagramio_query.txt [hunk 1]
```diff
@@ -8,23 +8,23 @@ Table USERS {
 
 Table RESTAURANTS {
   placeId varchar(300) [primary key]
-  name varchar(200) [not null]
-  address varchar(100) [not null]
+  name text [not null]
+  address text [not null]
```
Comment (yenting-biao) [inline db-docs/dbdiagramio_query.txt:12]: why do we changed to `text`?

File: src/db/schema.ts [hunk 2]
```diff
@@ -14,11 +14,11 @@ import {
 export const usersTable = pgTable(
   "users",
   {
-    userId: uuid("userId").defaultRandom().notNull().unique(),
-    ntuEmail: varchar("ntuEmail", { length: 63 }).notNull().unique(),
+    userId: uuid("userid").defaultRandom().notNull().unique(),
```
Comment (yenting-biao) [inline src/db/schema.ts:17]: What problems will we encounter if we use `userId` in the database?

File: db-docs/dbdiagramio_query.txt (hunk 1 above)
Comment (togimoto) [reply db-docs/dbdiagramio_query.txt:12]: Please look at the commit messages.

File: src/db/schema.ts (hunk 2 above)
Comment (togimoto) [reply src/db/schema.ts:17]: Please look at the commit messages.

File: db-docs/dbdiagramio_query.txt (hunk 1 above)
Comment (yenting-biao) [reply db-docs/dbdiagramio_query.txt]: > Please look at the commit messages.

我還沒看完先不要 resolve 吧
我剛剛想說 conversation 怎麼不見了

**Review by yenting-biao — APPROVED** [review]
ok

**Review by nukkk97 — APPROVED** [review]
nice

File: db-docs/dbdiagramio_query.txt (hunk 1 above)
Comment (togimoto) [reply db-docs/dbdiagramio_query.txt]: Sorry 😿

**Pull request merged**